plugins/plugin-hosted/codenvy-hosted-ext-hosted/src/main/java/com/codenvy/ide/hosted/client/inject/HostedEnvironmentGinModule.java

```diff
@@ -38,6 +37,6 @@ protected void configure() {
         bind(PromptToLoginView.class).to(PromptToLoginViewImpl.class);
         bind(BadConnectionNotifierView.class).to(BadConnectionNotifierViewImpl.class);
         bind(WorkspaceNotRunningView.class).to(WorkspaceNotRunningViewImpl.class);
-        bind(HostedWorkspaceStoppedHandler.class).asEagerSingleton();
+//        bind(HostedWorkspaceStoppedHandler.class).asEagerSingleton();
     }
```

Comment:
changes from this [commit](https://github.com/codenvy/codenvy/pull/716/commits/c9b018ecda0e220c2df0abef271fc44602a5b490)